### PR TITLE
Server bootstrapped

### PR DESCRIPTION
**Pull request by dbyington**
Setup the koa server with logging, routing, body, and cors.
Setup the router with the base routes for the app, login, and renew token.
Setup the basic service providers for Spotify and Duetify.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+'use strict';
+require('dotenv').config();
```
Comment (arol) [inline app.js:2]: genius

File: router.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+'use strict';
+const Router = require('koa-router');
+const router = new Router;
+
+const spotifyService = require('./service-providers/spotify-service');
+const duetifyService = require('./service-providers/duetify-service');
+
+router
+  .get('/login', spotifyService.login)
```
Comment (arol) [inline router.js:9]: Interesting, you name it as a service instead of as a controller 🤔

**Review by arol — APPROVED** [review]
Consider installing `eslint` as a dev dependency and setting it up in the future.

File: router.js (hunk 2 above)
Comment (dbyington) [reply router.js:9]: That's a good point. I'll note that for something to perhaps update, sooner rather than later.

**Pull request merged**